wasm/examples/node-sample/package.json

```diff
@@ -10,7 +10,7 @@
   "author": "Tomislav Markovski",
   "license": "ISC",
   "dependencies": {
-    "@trinsic/okapi": "1.0.0-preview.19"
+    "@trinsic/okapi": "file:../../packages/okapi"
```

Comment:
Is this guaranteed to be here?

Reply:
Only after a build, yes. The idea is for this sample to actually use the local version of the package, instead npm one.
This is why I made the `okapi-examples` repo which will use these packages as intended in production.